bin/hermit.hcl

```diff
@@ -1,4 +1,4 @@
 env = {
-  "CLOUDBEAT_VERSION": "8.12.0",
+  "CLOUDBEAT_VERSION": "8.13.0",
```

Comment:
I think this needs to be done separately because the snapshot builds for 8.13 are not available yet